OpenRA.Mods.Cnc/Traits/Infiltration/InfiltrateForTransform.cs

```diff
@@ -0,0 +1,57 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Mods.Common;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Cnc.Traits
+{
+	[Desc("This structure can be infiltrated causing funds to be stolen.")]
+	class InfiltrateForTransformInfo : ITraitInfo
+	{
+		[ActorReference, FieldLoader.Require] public readonly string IntoActor = null;
+		public readonly int ForceHealthPercentage = 0;
+		public readonly bool SkipMakeAnims = true;
+
+		public readonly BitSet<TargetableType> Types = default(BitSet<TargetableType>);
+
+		public object Create(ActorInitializer init) { return new InfiltrateForTransform(init, this); }
+	}
+
+	class InfiltrateForTransform : INotifyInfiltrated
```

Comment:
Can we name this `TransformOnInfiltration` to stay in line with `TransformOnCapture`?

Reply:
This would make it inconsistent with all the other `InfiltrateFor*` traits.